src/backend/collectibles.nim

```diff
@@ -0,0 +1,34 @@
+import json, json_serialization, strformat
+
+import ./core, ./response_type
+from ./gen import rpc
+
+type
+  NFTUniqueID* = ref object of RootObj
+    contractAddress* {.serializedFieldName("contract_address").}: string
+    tokenID* {.serializedFieldName("token_id").}: string
+
+proc `$`*(self: NFTUniqueID): string =
+  return fmt"""NFTUniqueID(
+    contractAddress:{self.contractAddress},
+    tokenID:{self.tokenID}
+  )"""
+
+proc `==`*(a, b: NFTUniqueID): bool = 
+  a.contractAddress == b.contractAddress and
+  a.tokenID == b.tokenID
```

Comment:
weird indent to have that one same as a normal new line

Reply:
fixed